src/components/UI/table.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { css } from 'emotion';
+import { node, bool, oneOfType, arrayOf, string } from 'prop-types';
+
+const TABLE = ({ children, zebra, ...props }) => {
+  const styles = css`
+    ${zebra ? 'tbody tr:nth-child(odd) {background-color: #e8e8e8;}' : ''};
+  `;
+  return (
+    <table className={styles} {...props}>
+      {children}
+    </table>
+  );
+};
+TABLE.propTypes = {
+  children: oneOfType([arrayOf(node), node]).isRequired,
+  zebra: bool,
+};
+TABLE.defaultProps = {
+  zebra: false,
+};
+
+const TBODY = ({ children, ...props }) => <tbody {...props}>{children}</tbody>;
```

Comment:
Oh default HTML elements don't offer proptypes?

Reply:
So I wanted to provide these wrappers so if we want to provide styling/functionality for basic DOM elements, we can do it all in one place and those changes will be reflected everywhere.